src/SprykerEco/Zed/Heidelpay/Business/Adapter/Mapper/ResponseFromHeidelpay.php

```diff
@@ -221,4 +222,18 @@ protected function mapPaymentFormUrl(
             $responseTransfer->setPaymentFormUrl(null);
         }
     }
+
+    /**
+     * @param \Heidelpay\PhpPaymentApi\Response $apiResponse
+     *
+     * @return string|null
+     */
+    protected function getConnectorInfo(Response $apiResponse): ?string
+    {
+        if ($apiResponse->getConnector()->getAccountIBan() === null) {
+            return null;
+        }
+
+        return $apiResponse->getConnector()->toJson();
```

Comment:
We need bank account information here. Connector object is alway exists because `getConnector()` method creates it with empty data in case if it wasn't created earlier. IBAN is main part of bank account information. So if `Connector` object has IBAN we convert it into json otherwise return null.